src/components/form/fields/schema-field.ts

```diff
@@ -260,4 +268,25 @@ export class SchemaField extends React.Component<FieldProps> {
 
         return React.createElement(JSONSchemaField, fieldProps);
     }
+
+    /**
+     * Gets the path to the current property within the schema
+     *
+     * @param {string} schemaId the id of the schema
+     * @returns {string[]} an array with the schema path for the current property
+     * @example
+     * const schemaId = 'root_sections_0_controls_0_name';
+     * const schemaPath = getSchemaPath(schemaId);
+     * // => ['sections', '0', 'controls', '0', 'name']
+     *
+     * The result is compatible with the Lodash `get` function
```

Comment:
Not sure if we need to mention this since it might create a weird dependency on Lodash. If they change their API this will be outdated unless we also change it.

Reply:
Removed!